Library/Homebrew/test/Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "mocha", "~> 1.1"
 gem "minitest", "~> 5.3"
 gem "rake", "~> 10.3"
+gem "test-unit", "~> 3.2"
```

Comment:
:+1:, but we'll have to remember to update this periodically unless we send this patch upstream.